gwpy/signal/qtransform.py

```diff
@@ -515,6 +518,9 @@ def interpolate(self, tres=.001, fres="<default>",
 
         Notes
         -----
+        This method will return a `Spectrogram` of dtype ``float32``
```

Comment:
```suggestion
        This method will return a `Spectrogram` of dtype ``float32`` if
```

Reply:
thanks @duncanmmacleod, I've committed this change.